src/commands/rm.hpp

```diff
@@ -17,11 +17,23 @@ class RmCommand : public liteshell::BaseCommand
         for (auto &target : context.values.at("targets"))
         {
             auto targets = utils::list_files(target);
+            if (targets.empty())
+            {
+                auto message = utils::format("Error: Target \"%s\" does not exist or is empty", target.c_str());
```

Comment:
`utils::list_files` also returns empty directories.
```suggestion
                auto message = utils::format("Warning: Target \"%s\" does not exist", target.c_str());
```